scripts/generate_records/package.json

```diff
@@ -41,4 +41,4 @@
     "@cumulus/api-client": "18.5.0",
```

Comment:
```suggestion
    "@cumulus/api-client": "18.5.1",
```

Reply:
I'm confused by this suggestion - api client *is* being updated to 18.5.1 already in the PR

Reply:
Talked about this in a hangout, going to remove the extra line from the dev dependencies.

Reply:
afde3d0e3cd